build.gradle.kts

```diff
@@ -965,9 +982,11 @@ allprojects {
                             asNode()
                         }
                         name.set(
-                            (project.findProperty("artifact.name") as? String) ?: "Calcite ${project.name.capitalize()}"
+                            (project.findProperty("artifact.name") as? String) ?: "Calcite ${project.name.replaceFirstChar {
```

Comment:
For our context I think that `String.replaceFirstChar { it.uppercaseChar() }` should be sufficient. We don't have exotic characters in the project name.

Also maybe worth mentioning in the commit message that we replace `capitalize` cause it is deprecated.